packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -619,6 +629,21 @@ export async function activate(context: vscode.ExtensionContext) {
   telemetryService.sendExtensionActivationEvent(extensionHRStart);
   console.log('SFDX CLI Extension Activated');
   return api;
```

Comment:
@jag-sfdc Is this return in the proper place?

Reply:
@rcoringrato-sfdc - Definitely not. Fixing it now.